src/libs/elektra/key.c

```diff
@@ -733,13 +757,18 @@ int keyLock (Key * key, elektraLockFlags what)
 }
 
 /**
- * @brief Tests if a part of a key is locked
+ * Tests if parts of a Key are locked
```

Comment:
It returns concrete information, so maybe something like:
```suggestion
 * Checks which parts of a Key are locked
```